sources/packages/web/src/services/StudentAppealService.ts

```diff
@@ -1,8 +1,9 @@
 import ApiClient from "@/services/http/ApiClient";
-import { StudentAppealDTO } from "@/types/contracts/student/StudentRequestChange";
+import { StudentAppealRequest } from "@/types";
 import {
-  StudentAppealApiOutDTO,
-  StudentAppealRequestApiInDTO,
+  StudentAppealAPIOutDTO,
+  StudentAppealRequestAPIInDTO,
+  StudentAppealRequestApprovalAPIInDTO,
 } from "./http/dto/StudentAppeal.dto";
```

Comment:
Not a part of PR but as DTOs changed import can be modified to use @

Reply:
Changed!